ros/src/tl_detector/tl_detector.py

```diff
@@ -14,7 +14,7 @@
 import math
 
 STATE_COUNT_THRESHOLD = 3
-TL_LOOK_AHEAD = 50
+TL_LOOK_AHEAD = 100
```

Comment:
What's the impact of this change ?

Reply:
I should have put a comment there..
This defines how far the car sees traffic lights.
It the distance is too small, the car will suddenly find a traffic light, but will not have enough time to stop. Having too big distance results in extra work our tl_detector node should do.

Reply:
I've also changed it to 100. So after rebase we see something like this:
```
- TL_LOOK_AHEAD =100 
+ TL_LOOK_AHEAD = 100
```